src/plugin.yml

```diff
@@ -1,8 +1,8 @@
 name: CS-CoreLib
-version: ${project.version}
-authors: [mrCookieSlime, John000708, AlexLander123]
+version: 5
```

Comment:
Do not modify the version like that, let Maven handle that.

Reply:
kk